==== //depot/qt/3/src/kernel/qtextengine.cpp#32 - /home/lars/dev/qt-3/src/kernel/qtextengine.cpp ====
--- /tmp/tmp.4824.0	2005-01-14 09:47:21.702603728 +0100
+++ /home/lars/dev/qt-3/src/kernel/qtextengine.cpp	2005-01-14 09:47:20.457792968 +0100
@@ -829,6 +829,8 @@
         int tcls = ncls;
         if (tcls >= QUnicodeTables::LineBreak_SA)
             tcls = QUnicodeTables::LineBreak_ID;
+        if (cls >= QUnicodeTables::LineBreak_SA)
+            cls = QUnicodeTables::LineBreak_ID;
 
 	int brk = charAttributes[i].charStop ? breakTable[cls][tcls] : (int)Pbk;
 #endif
